metadata-ingestion/src/datahub/cli/check_cli.py

```diff
@@ -163,6 +163,27 @@ def plugins(source: Optional[str], verbose: bool) -> None:
     )
 
 
+@check.command()
+@click.option(
+    "--sql",
```

Comment:
Would be cool if this supported a file. Or could you do this with something like `cat <file_name> | datahub check sql_format --platform snowflake --sql`?

Reply:
I often use it like this `datahub check sql-format  --platform snowflake --sql  "$(< filename.sql)"`

but agree that it's a bit cumbersome - we can revisit this in the future